app/controllers/members_controller.rb

```diff
@@ -6,6 +6,8 @@ def destroy
     toggle_active_posts
     @member.destroy
 
+    OrganizationNotifier.member_deleted(@member).deliver_now
```

Comment:
Just curious, why `deliver_now` instead of `deliver_later`?

Reply:
Right! I just copy-pasted the same workflow we are already using for emails, but I think this is probably something from the past. I'll move all email delivery to the background queues.

Reply:
Migrated to `deliver_later`.